.circleci/config.yml

```diff
@@ -8,30 +8,30 @@ jobs:
     executor: node/default
     steps:
       - checkout
-      - restore_cache:
-          name: Restore pnpm Package Cache
-          keys:
-            - pnpm-packages-{{ checksum "pnpm-lock.yaml" }}
       - run:
-          name: Install pnpm package manager
-          command: |
-            corepack enable
-            corepack prepare pnpm@latest-9 --activate
-            pnpm config set store-dir .pnpm-store
+          name: Install dependencies
+          command: npm install
       - run:
-          name: Install Dependencies
-          command: |
-            pnpm install
-      - save_cache:
-          name: Save pnpm Package Cache
-          key: pnpm-packages-{{ checksum "pnpm-lock.yaml" }}
-          paths:
-            - .pnpm-store
-      - run:
-          command: pnpm test
           name: Run tests
+          command: npm run test
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: npm install
+      - run:
+          name: Build
+          command: npm run build
 workflows:
   test_my_app:
     jobs:
       - test
+  test_and_build_my_app:
```

Comment:
is it testing and building the app or just building?

Reply:
test and build, the build job requires the test job in the workflow

Reply:
Then I think you can change this name to only build, wdyt?